packages/protocol/contracts/stability/StableToken.sol

```diff
@@ -461,47 +462,74 @@ contract StableToken is
   }
 
   /**
-   * @notice Deduct balance for making payments for gas in this StableToken currency.
-   * @param from The account to debit balance from
-   * @param value The value of balance to debit
+   * @notice Transfers StableToken from one address to another
+   * @param to The address to transfer StableToken to.
+   * @param value The amount of StableToken to be transferred.
+   */
+  function _transfer(address to, uint256 value) internal returns (bool) {
```

Comment:
who uses this function? why is it changed on this PR?

Reply:
Just some weirdness with diffs, there are no real changes.